### ISSUE TITLE
Disable accessibility features altogether

### ISSUE THREAD

**Issue by 10538** [phonegap/phonegap-mobile-accessibility#6]
I have a PhoneGap Build app that includes its own audio output. Unfortunately, the inbuilt accessibility features of Android and iOS disrupt the handling of touch events, preventing the app from functioning properly. It is a word game that involves users making words by dragging their finger across the screen. The audio mode speaks the letters as they do this. However, with the device's accessibility mode enabled, dragging is no longer possible.

I would dearly like to disable the capture of touch events by the inbuilt accessibility features. As it is, I have to tell users that they cannot enable audio mode on the app if they have accessibility features switched on. I have tried using the 'aria-ignore' attribute in the enclosing div for the game page, but to no effect. 

The standard app (without the audio feature) can be found on Google Play or the App Store by searching for 'wordeke'.
